zigpy_cli/radio.py

```diff
@@ -25,9 +25,10 @@
 @click.argument("radio", type=click.Choice(list(RADIO_TO_PACKAGE.keys())))
 @click.argument("port", type=str)
 @click.option("--baudrate", type=int, default=None)
+@click.option("--flow_control", type=int, default=None)
```

Comment:
```suggestion
@click.option("--flow-control", type=click.Choice(["software", "hardware", "none"]), default="none")
```

Reply:
ah yeah, the setting with the underscore, but the argument probably should be with the hyphen.